### PR TITLE
Upgrade Karma config to 0.10 format

### LINKED ISSUE #19: Upgrade to Karma 0.10

### PR DESCRIPTION
**Pull request by passy**
Fixes #19

I had to drop es5sh[ia]m from the inclusion list, because I couldn't figure out
how to load them before Require.js is initialized. Since Require.js is now a
framework plugin you can no longer specify a certain point in the inclusion
chain where it should be loaded. Because es5-shim has an UMD header, it tries to
use `define()` which causes an error when invoked from outside of the
Require.js loading cycle.

Any ideas how this could be re-enabled?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,14 +25,15 @@
...
+    "karma": ">=0.10.0",
     "yo": ">=1.0.0-rc.1.1"
   },
   "devDependencies": {
     "mocha": "~1.8.1"
   },
   "engines": {
-    "node": ">=0.8.0"
+    "node": ">=0.8.0",
+    "npm": ">=1.2.10"
```
Comment (passy) [inline package.json:36]: Added this because peer dependencies were used which are supported since npm 1.2.10.

**Comment by necolas** [comment]
See this for how it needs to be done: https://github.com/flightjs/flight/commit/4ebfa318d1478850e01f64f99746db1accbcd860

You also need to add the launcher plugins as dependencies, and please remove defaults like `loglevel`, `port`, etc.

**Comment by passy** [comment]
Didn't think about looking at the flight main repo. Updated the config and dependencies accordingly.

File: package.json [hunk 2]
```diff
@@ -25,14 +25,15 @@
     "yeoman-generator": "~0.12.2"
   },
   "peerDependencies": {
-    "karma": ">=0.8.5",
+    "karma": ">=0.10.0",
```
Comment (necolas) [inline package.json]: Karma will defer to running the local installation, so we'll get greater coverage by leaving it at `0.8.x` for now.

Comment (passy) [inline package.json]: Good point. Updated.

**Comment by necolas** [comment]
Thanks. Applied in 21a8608e8aab2983f2a3c0025628b95e48521758

**Pull request closed without merging**